ztest/ztest.go

```diff
@@ -548,18 +552,20 @@ func runzq(path, zed, input string, outputFlags []string) (string, string, error
 	if err != nil {
 		return "", "", err
 	}
+	var outflags outputflags.Flags
+	var inflags inputflags.Flags
+	var flags flag.FlagSet
+	outflags.SetFlags(&flags)
+	inflags.SetFlags(&flags)
+	if err := flags.Parse(append(outputFlags, inputFlags...)); err != nil {
```

Comment:
Using a single `flag.FlagSet` like this allows output flags in the `input-flags` field and input flags in the `output-flags` field, which feels wrong, but maybe it isn't.

Reply:
Yeah that's a good call, will fix.